<launch>
    <param name="map_file" value="/home/mwisse/mirte_ws/src/map_interaction/maps/blank.yaml" />
     
    <node name="map_switcher_service" pkg="map_interaction" type="map_switcher_service.py" output="screen"/>
    <node name="map_store_pose_service" pkg="map_interaction" type="map_store_pose_service.py" output="screen"/>
    <node name="map_switch_interactive" pkg="map_interaction" type="map_switch_interactive.py" output="screen"/>
    <node name="map_save_interactive"   pkg="map_interaction" type="map_save_interactive.py"   output="screen"/>
    <node name="marker_publisher_node" pkg="map_interaction" type="marker_publisher_node.py" output="screen"/>
    <node name="laser_filter" pkg="map_interaction" type="laser_filter.py" output="screen"/>
    <node name="inspect_holonomic_action_server" pkg="map_interaction" type="inspect_holonomic_action_server.py" output="screen"/>
</launch>
